langroid/agent/special/neo4j/neo4j_chat_agent.py

```diff
@@ -87,12 +87,13 @@ def __init__(self, config: Neo4jChatAgentConfig):
         self._import_neo4j()
         self._initialize_db()
         self._init_tools_sys_message()
-        self.init_state()
+        # self.init_state() # already called inside _init_tools_sys_message()
```

Comment:
we should leave it in here, since we may call `init_state()` to reset state after the agent has been initialized.